sdk/src/realtime/composable.ts

```diff
@@ -243,19 +246,27 @@ export function realtime(config: WebSocketConfig = {}) {
 					let resolved = false;
 					const ws = new client.globals.WebSocket(url);
 
+					const timeout = setTimeout(
```

Comment:
What about if the user sets config.connect = undefined on purpose? Maybe we should wrap it in an if?

Reply:
That is covered by L253 which defaults to `1000`

Reply:
Good point, added in bdd67e1821673ae21eaedafd534e17ff0d0ffbd8